cfme/storage/volume.py

```diff
@@ -243,6 +260,22 @@ def create_snapshot(self, name, cancel=False, reset=False):
                 view.save.click()
                 return snapshot_collection.instantiate(name, self.provider)
 
+
+    def attach_instance(self, name, mountpoint=None):
+        view = navigate_to(self, 'AttachInstance')
+        view.fill({'instance': name})
+        view.fill({'mountpoint': mountpoint})
```

Comment:
Required:
Jad please use `.fill` for view directly no need to use individual widget `fill` method...
like
```
view.fill({'instance': name, 'mountpoint': mountpoint})
```